runtime/src/main/java/io/quarkus/logging/sentry/SentryHandlerValueFactory.java

```diff
@@ -76,6 +76,8 @@ public static SentryOptions toSentryOptions(SentryConfig sentryConfig) {
             });
         }
 
+        sentryConfig.ignoredErrors.ifPresent(options::setIgnoredErrors);
```

Comment:
shouldn't this be 3 lines below and not wrapped in blank lines?

Reply:
I put the line for `ignoredErrors` below the existing block for `ignoredExceptionsForType`, but I can put it anywhere you want.
Which line do you want me to move it to?

Reply:
i think it would look better on line 82 where a whole bunch of `ifPresent` checks are.

Reply:
I've committed a change in this regard.